### PR TITLE
MaterialDesignMultiFloatingActionPopupBox RTL bug fixed

### PR DESCRIPTION
**Pull request by hshahdoost**
I tried to make them as small as possible, this one fixes the FloatingActionButton popup's bug on RTL

### CODE REVIEW

File: MaterialDesignThemes.Wpf/PackIcon.cs [hunk 1]
```diff
@@ -17,9 +17,11 @@ public class PackIcon : PackIconBase<PackIconKind>
         static PackIcon()
         {
             DefaultStyleKeyProperty.OverrideMetadata(typeof(PackIcon), new FrameworkPropertyMetadata(typeof(PackIcon)));
-        }     
+        }
 
         public PackIcon() : base(PackIconDataFactory.Create)
-        { }    
+        {
+            FlowDirection = FlowDirection.LeftToRight;
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: What is this change for?

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.CheckBox.xaml [hunk 2]
```diff
@@ -82,7 +82,8 @@
                             <ColumnDefinition Width="Auto"/>
                             <ColumnDefinition Width="*"/>
                         </Grid.ColumnDefinitions>
-                        <Viewbox Width="18" Height="18"  VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
+                        <Viewbox Width="18" Height="18"  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                                 FlowDirection="LeftToRight">
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.CheckBox.xaml]: ? hardcoding?

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml [hunk 3]
```diff
@@ -136,7 +136,7 @@
 						</VisualStateManager.VisualStateGroups>
 						<Ellipse Fill="{TemplateBinding Background}" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" x:Name="UncheckedEllipse" Opacity=".54" />
 						<Ellipse Fill="{TemplateBinding Background}" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" x:Name="CheckedEllipse" />
-						<Viewbox Width="24" Height="24" x:Name="CheckMark" Visibility="Hidden" HorizontalAlignment="Center" VerticalAlignment="Center">
+						<Viewbox Width="24" Height="24" x:Name="CheckMark" Visibility="Hidden" HorizontalAlignment="Center" VerticalAlignment="Center" FlowDirection="LeftToRight">
```
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml]: ? hardcoding?

**Comment by ButchersBoy** [comment]
Please see comments, looks like we have hard coded in a couple of places?  Shouldn't we be TemplateBinding them in?

File: MaterialDesignThemes.Wpf/PackIcon.cs (hunk 1 above)
Comment (hshahdoost) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: Icons are mirrored in RTL mode. which is not exactly the way they were meant to be shown. So I had to give them LTR FlowDirection directly

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.CheckBox.xaml (hunk 2 above)
Comment (hshahdoost) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.CheckBox.xaml]: The same thing as Icons happen here. the check mark inside the Checkbox should look the same on RTL or LTR.

File: MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml (hunk 3 above)
Comment (hshahdoost) [inline MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ToggleButton.xaml]: Exactly the same as Icons and checkboxes, The Checked mark inside toggle button should always look the same

File: MaterialDesignThemes.Wpf/PackIcon.cs (hunk 1 above)
Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: OK, I understand, thanks.

Comment (punker76) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: But this should be done in the theme style, so it's even possible to change it...

Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: @punker76 good point.
@hshahdoost can you try adding this around here instead? https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/Generic.xaml#L789

**Comment by hshahdoost** [comment]
This is how checkboxes are in RTL, it's a WPF bug which mirrors the pictures in RTL mode and should be changed to LTR. It happens in three places Icons, ToggleButtons and Checkboxes

checkboxes

File: MaterialDesignThemes.Wpf/PackIcon.cs (hunk 1 above)
Comment (hshahdoost) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: @punker76 @ButchersBoy you are right. I should move it to Generic.xaml.
Do I have to revert it in here? or make another commit?

Comment (ButchersBoy) [inline MaterialDesignThemes.Wpf/PackIcon.cs]: Yeah, delete it, add the new code in, commit, push, all on the same branch.  The beauty of code reviews with GitHub...you can just keep working on this PR until everything is code reviewed, then I will merge it.

**Comment by ButchersBoy** [comment]
:+1:

**Comment by hshahdoost** [comment]
Thanx a lot.
